shiny/ui/_theme_brand.py

```diff
@@ -0,0 +1,297 @@
+from __future__ import annotations
+
+import re
+import warnings
+from pathlib import Path
+from typing import Any, Optional
+
+from brand_yml import Brand
+from htmltools import HTMLDependency
+
+from .._versions import bootstrap
+from ._theme import Theme
+from ._theme_presets import ShinyThemePreset, shiny_theme_presets
+
+color_map = {
+    "foreground": ["brand--foreground", "body-color", "pre-color"],
+    "background": ["brand--background", "body-bg"],
+    "primary": ["primary"],
+    "secondary": ["secondary", "body-secondary-color", "body-secondary"],
+    "tertiary": ["body-tertiary-color", "body-tertiary"],
+    "success": ["success"],
+    "info": ["info"],
+    "warning": ["warning"],
+    "danger": ["danger"],
+    "light": ["light"],
+    "dark": ["dark"],
+}
+"""Maps brand.color fields to Bootstrap Sass variables"""
+
+bootstrap_colors = {
+    # https://github.com/twbs/bootstrap/blob/6e1f75/scss/_variables.scss#L38-L49
+    "5": [
```

Comment:
Just curious, why scope these (and not the others like `color_map` and `typography_map`) under the version?

Reply:
This part seemed more likely to change between versions and I was using it as a way to start to think about Bootstrap versioning. I'm going to remove it though. We can deal with versioning when we actually have more than one Bootstrap version to choose from (or _if we have_...)

Reply:
Agreed, this is the way I'd do it (later, not in this PR)